tests/managed_services/mas_debezium/conftest.py

```diff
@@ -0,0 +1,199 @@
+    ServiceAccountCreateRequestData,
+)
+
+from utilities.template_utils import render_template_from_dict
+
+
+LOGGER = logging.getLogger(__name__)
+WAIT_STATUS_TIMEOUT = 120
+
+
+@pytest.fixture(scope="class")
+def kafka_instance(kafka_mgmt_api_instance):
+    _async = True
+    kafka_request_payload = KafkaRequestPayload(
+        cloud_provider="aws",
+        name=KAFKA_NAME,
+        region="us-east-1",
+        plan="standard.x1",
+        reauthentication_enabled=True,
+    )
+    kafka_create_api = kafka_mgmt_api_instance.create_kafka(
+        _async=_async, kafka_request_payload=kafka_request_payload
+    )
+    assert (
+        kafka_create_api.status == "accepted"
+    ), f"Failed to create a kafka instance. API response:\n{kafka_create_api}"
+
+    yield kafka_create_api
+
+    kafka_mgmt_api_instance.delete_kafka_by_id(
+        async_req=True, _async=_async, id=kafka_create_api.id
+    )
+
+
+@pytest.fixture(scope="class")
+def kafka_instance_ready(kafka_mgmt_api_instance, kafka_instance):
+    try:
+        kafka_status_samples = TimeoutSampler(
+            wait_timeout=360,
+            sleep=10,
+            func=lambda: kafka_mgmt_api_instance.get_kafka_by_id(
+                id=kafka_instance.id
+            ).status
+            == "ready",
+        )
+        for sample in kafka_status_samples:
+            if sample:
+                break
+        kafka_ready_api = kafka_mgmt_api_instance.get_kafka_by_id(id=kafka_instance.id)
+        LOGGER.info(f"Kafka instance is ready:\n{kafka_ready_api}")
+        yield kafka_ready_api
+    except TimeoutExpiredError:
+        LOGGER.error(
+            f"Timeout expired. Kafka creation status: "
+            f"{kafka_mgmt_api_instance.get_kafka_by_id(id=kafka_instance.id).status}"
+        )
+        raise
+
+
+@pytest.fixture(scope="class")
+def kafka_instance_client(kafka_instance_ready, access_token):
+    # https://github.com/redhat-developer/app-services-sdk-python/tree/main/sdks/kafka_instance_sdk
+    configuration = rhoas_kafka_instance_sdk.Configuration(
+        host=kafka_instance_ready.admin_api_server_url, access_token=access_token
+    )
+
+    with rhoas_kafka_instance_sdk.ApiClient(
+        configuration=configuration
+    ) as kafka_api_client:
+        yield kafka_api_client
+
+
+@pytest.fixture(scope="class")
+def kafka_instance_sa(kafka_instance_client, service_accounts_api_instance):
+    service_account_create_request_data = ServiceAccountCreateRequestData(
+        name="ms-kafka-sa",
+        description=f"{KAFKA_NAME} instance service-account",
+    )
+    kafka_sa = service_accounts_api_instance.create_service_account(
+        service_account_create_request_data=service_account_create_request_data
+    )
+    assert (
+        kafka_sa.id
+    ), f"Failed to create service-account for kafka. API response:\n{kafka_sa}"
+    LOGGER.info(f"kafka service-account:\n{kafka_sa}")
```

Comment:
why is this log needed?

Reply:
only for info on the kafka's service account as its a created resource.
removing.